### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by shharrnam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Apache Spark
+# Apache Spark 2
```
Comment (shharrnam) [inline README.md:1]: Commenting

**Review by shharrnam — COMMENTED** [review]
Commenting

**Comment by shharrnam** [comment]
Commenting on MR

**Comment by shharrnam** [comment]
> Commenting on MR

Reply to comment on MR

File: README.md (hunk 1 above)
Comment (shharrnam) [reply README.md:1]: Reply to comment on code

**Review by shharrnam — COMMENTED** [review]
Final Review Comment

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 
 Spark is a unified analytics engine for large-scale data processing. It provides
```
Comment (shharrnam) [inline README.md:3]: Code comment

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
...
 high-level APIs in Scala, Java, Python, and R, and an optimized engine that
```
Comment (shharrnam) [inline README.md:4]: Code comment 2

File: README.md (hunk 2 above)
Comment (shharrnam) [inline README.md:3]: Code comment 3

File: README.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Apache Spark
```
Comment (shharrnam) [inline README.md:1]: why?

Comment (shharrnam) [inline README.md:1]: why2?

**Review by shharrnam — COMMENTED** [review]
why2 heading

File: README.md (hunk 4 above)
Comment (shharrnam) [inline README.md:1]: why3?

File: README.md (hunk 2 above)
Comment (shharrnam) [reply README.md:3]: reply to code comment3

File: README.md (hunk 1 above)
Comment (shharrnam) [inline README.md:1]: Comment 1 with review

File: README.md (hunk 3 above)
Comment (shharrnam) [inline README.md:4]: Comment 2 with review

**Review by shharrnam — COMMENTED** [review]
Top comment on review

File: README.md (hunk 4 above)
Comment (shharrnam) [inline README.md:1]: Comment3 with review

File: README.md (hunk 1 above)
Comment (shharrnam) [inline README.md:1]: Comment 4 with review

File: README.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
...
+# Apache Spark 2
```
Comment (shharrnam) [inline README.md:2]: Comment 5 with review

Comment (shharrnam) [inline README.md:2]: Comment6 without review

File: README.md (hunk 3 above)
Comment (shharrnam) [reply README.md:4]: Reply to comment2

**Review by shharrnam — COMMENTED** [review]
Review Comment - Starting a review for Breejesh to comment

**Comment by shharrnam** [comment]
Top level comment

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (breejesh2212) [reply README.md:1]: Added a comment

**Comment by shharrnam** [comment]
Top level comment 2

File: README.md (hunk 4 above)
Comment (dharinp) [inline README.md:1]: Hi hello

**Review by dharinp — CHANGES_REQUESTED** [review]
Hi hello request change

File: README.md (hunk 4 above)
Comment (dharinp) [inline README.md:1]: Testing 2

**Review by dharinp — APPROVED** [review]
This is approve testing

**Review by dharinp — APPROVED** [review]

**Comment by shharrnam** [comment]
Closing MR With comment

**Pull request closed without merging**